internal/cmd/envs.go

```diff
@@ -401,6 +412,16 @@ func buildUpdateReq(ctx context.Context, client *coder.Client, conf updateConf)
 		defaultDiskGB   int
 	)
 
+	if conf.useCVM && conf.noCVM {
+		return nil, xerrors.New("--container-vm and --not-container-vm flags conflict")
+	}
+	if conf.useCVM {
+		updateReq.UseContainerVM = boolP(true)
+	}
+	if conf.noCVM {
+		updateReq.UseContainerVM = boolP(false)
+	}
+
```

Comment:
Scenario:

I have an existing CVM enabled env.

I wan't to update memory.

I don't wan't to disable CVM feature on said env but if I don't pass `--container-vm` again the update will assume that I do?

Does this mean I have to pass `--container-vm` every time I wan't to make an environment update but not toggle using the CVM itself?

If so, I believe this burden should fall on the program and not the user. 

It's not fair to force the user to be explicit or have to remember special details like this.

My understanding is that when we attempt to take an env from cvm-enabled -> cvm-disabled and vice versa the environment fails to rebuild successfully anyway ( correct me if i'm wrong )

If that's correct, should we be allowed to toggle this on env updates at all?

Reply:
I think the previous comments should resolve these concerns?

Reply:
I understand the partial update. It's not obvious to me why we are allowing the user to toggle enabling cvm on an env if that will fail the rebuild?

Reply:
Ah I see, this will not be the behavior shortly. We're going to have upgrading/downgrading working properly shortly.

Reply:
Ah ok well that's good.

What do you think about a single flag for toggling state where if you use it the value of `UseCVM` becomes `!env.UseCVM` and maybe prompt the user about the implications of switching or even prompt for confirmation? Trying to think of alternatives

Reply:
Toggle flags are not commonly used for cases like this as it can cause super odd behavior based on prior state. Running the same edit command twice though yield the same result.

Having a positive and negative flag, although not ideal, feels like the correct approach here.

Reply:
> If so, I believe this burden should fall on the program and not the user.

The partial update behavior takes care of this...